src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -190,14 +197,14 @@ public bool TryFormat(Span<char> destination, out int charsWritten, ReadOnlySpan
         }
 
         //
-        // IConvertible implementation
+        // IConvertible
```

Comment:
Consider using #region to organize methods?

Reply:
We tend to not use regions in the libraries code base. Many devs seem to have a strong aversion to them ☹️